pkg/ddl/index.go

```diff
@@ -862,32 +878,6 @@ func cleanupSortPath(ctx context.Context, currentJobID int64) error {
 	return nil
 }
 
-// IngestJobsNotExisted checks the ddl about `add index` with ingest method not existed.
```

Comment:
Why here deleted this func?

Reply:
This func is used when log backup is not compatible with fast create index. In 7.0+, it's useless.